### DISCUSSION TITLE
How to deal with unknown message ordering in kafka

### DISCUSSION THREAD

**Discussion by ethrbunny** [ovh/venom#687] [Q&A] [answered]
Working with a kafka consumer. Its problematic to do assertions as messages will be produced in unknown order. 

IE if  I produce three messages with "value": "one", "two", or "three" I can't assert 

result.messagesjson.messagesjson0.value ShouldEqual "one" 

when it might be messagejson0, 1, or 2 containing that "value".

Wondering if there is a way to work this out.

**Comment by fokion** [comment, marked as answer]
Is [this](https://github.com/ovh/venom#using-logical-operators) something that you can use  ?
